.github/stale.yml

```diff
@@ -0,0 +1,40 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale (approx. 9 months)
+daysUntilStale: 270
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually but will remain marked as stale.
+daysUntilClose: 7
+
+# Only issues or pull requests with all of these labels are check if stale. Defaults to `[]` (disabled)
+onlyLabels: []
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels:
+  - pinned
+
+# Set to true to ignore issues in a project (defaults to false)
+exemptProjects: false
+
+# Set to true to ignore issues in a milestone (defaults to false)
+exemptMilestones: false
+
+# Set to true to ignore issues with an assignee (defaults to false)
+exemptAssignees: false
+
+# Label to use when marking as stale
+staleLabel: stale
+
+# Comment to post when marking as stale. Set to `false` to disable
+markComment: >
+  Hey Friends, this issue has been automatically marked as `stale` because it has not had
+  recent activity. It will be closed if no further activity occurs. Please add the 
+  `pinned` label if you feel that this issue needs to remain open/active. 
+  Thank you for your contributions and help in keeping things tidy!
+
+# Limit the number of actions per hour, from 1-30. Default is 30
+limitPerRun: 30
+
+# Limit to only `issues` or `pulls`
+only: issues
```

Comment:
Would it be worth adding `pulls` here, too?

Reply:
I'd like to parse through the pulls a bit more given that folks put in the extra effort to make the changeset and I'd like to validate if the changes are still relevant before something beneficial gets closed.